deployment/ds-hostnet-split/03-envoy.yaml

```diff
@@ -18,8 +18,8 @@ spec:
       annotations:
         prometheus.io/scrape: "true"
         prometheus.io/port: "8002"
-        prometheus.io/path: "/stats"
-        prometheus.io/format: "prometheus"
+        prometheus.io/statsdport: "9102"
+        prometheus.io/path: "/stats/prometheus"
```

Comment:
Am I missing something or this will scrape only envoy metrics? stastd, I think, will gather only envoy and so will port `8002`. Right?

I think metrics for the contour container (on endpoint `/metrics` and port `8000`) are not gathered by this. Am I missing something? Hopefully I am :)

I will try to create a cluster and see if those are missing or not, it's too late here now to try this :-(

Reply:
@rata Contour metrics are scraped on the Contour deployment. This example is applied to the split model where Contour & Envoy are not colocated in the same pod.

Reply:
Ohh, I see. Silly me, thanks! :)